utils/test/jest.config.js

```diff
@@ -26,8 +26,9 @@ module.exports = {
     '\\.(svg)$': '<rootDir>/utils/test/svg-mock.js'
   },
   collectCoverageFrom: [
-    '<rootDir>/packages/*!(.template)/src/**/*.{js,jsx}',
+    '<rootDir>/packages/*/src/**/*.{js,jsx}',
```

Comment:
My addition of the `index.spec.js` tests broke our code coverage 😢 This re-enables that feature